docs/source/contributing.md

```diff
@@ -0,0 +1,87 @@
+# Contributing to Scylla Operator
```

Comment:
We should either move it out of the doc tree and make it a standalone file (something like CONTRIBUTING.md at the top level) or add it to the index (it would require a new subtree I suppose, there doesn't seem to be a fitting one atm)

Reply:
+1, I think we should stick to `CONTRIBUTING.md` in root directory only.

Reply:
I feel the same way. Glad to see this comment. Done.